inline.h

```diff
@@ -291,6 +291,28 @@ S_strip_spaces(pTHX_ const char * orig, STRLEN * const len)
 }
 #endif
 
+/* ------------------------------- hv.c ------------------------------- */
+
+/* Common code for hv_delete()/hv_exists()/hv_fetch()/hv_store()  */
+void *
```

Comment:
Missing `PERL_STATIC_INLINE`

Though I expect the compilers picked up the inline-edness from proto.h